README.md

```diff
@@ -369,4 +370,242 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
+#	127.0.0.1       localhost
+#	::1             localhost
+
+# Added by Docker for Windows
+10.244.50.193 host.docker.internal
+10.244.50.193 gateway.docker.internal
+# End of section
+
+<Your ipv4 address> localhost-k8s
+
+```
+
+Note: Make sure to leave a new line at the end before saving it. 
+
+### 11. Install alfresco-dbp
+
+Copy and paste the following block into your command line, making sure to replace <yourNamespace> accordingly. 
+  
+```bash
+helm install alfresco-incubator/alfresco-dbp ^
+--set alfresco-content-services.externalHost="localhost-k8s" ^
+--set alfresco-content-services.networkpolicysetting.enabled=false ^
+--set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://localhost-k8s/auth" ^
+--set alfresco-content-services.repository.replicaCount=1 ^
+--set alfresco-content-services.repository.livenessProbe.initialDelaySeconds=420 ^
+--set alfresco-content-services.pdfrenderer.livenessProbe.initialDelaySeconds=300 ^
+--set alfresco-content-services.libreoffice.livenessProbe.initialDelaySeconds=300 ^
+--set alfresco-content-services.imagemagick.livenessProbe.initialDelaySeconds=300 ^
+--set alfresco-content-services.share.livenessProbe.initialDelaySeconds=420 ^
+--set alfresco-content-services.repository.resources.requests.memory="2000Mi" ^
+--set alfresco-content-services.pdfrenderer.resources.requests.memory="500Mi" ^
+--set alfresco-content-services.imagemagick.resources.requests.memory="500Mi" ^
+--set alfresco-content-services.libreoffice.resources.requests.memory="500Mi" ^
+--set alfresco-content-services.share.resources.requests.memory="1000Mi" ^
+--set alfresco-content-services.postgresql.resources.requests.memory="500Mi" ^
+--set alfresco-process-services.processEngine.environment.IDENTITY_SERVICE_AUTH="http://localhost-k8s/auth" ^
+--set alfresco-process-services.processEngine.resources.requests.memory="1000Mi" ^
+--set alfresco-process-services.adminApp.resources.requests.memory="250Mi" ^
+--namespace <yourNameSpace>
+```
+
+Repeatedly  run the following command until you can see that all the pods are successfully installed. This can take up to one hour. 
+
+```bash
+kubectl get pods --namespace ayman
+```
+
+If any pods are failing, you can use each of the following commands to see more about their errors:
+
+```bash
+kubectl logs <the part of the pod name that all the pods have in common> --namespace <your namespace name>
+kubectl describe pod <the part of the pod name that all the pods have in common> --namespace <your namespace name>
+```
+
+If the postgres pod won't start and you are getting the following error:
+
+```bash
+2019-02-08 10:27:04.358 UTC [1] FATAL:  data directory "/var/lib/postgresql/data/pgdata" has wrong ownership
+2019-02-08 10:27:04.358 UTC [1] HINT:  The server must be started by the user that owns the data directory.
+```
+
+Run the following commands to delete the storageClass hostpath and set up the hostpath provisioner: 
+
+```bash
```

Comment:
Should we move this before installing the DBP chart? Since these instructions are for windows all the users will encounter the issue with the hostpath provisioner